### PR TITLE
Feat/reports header

### PR DESCRIPTION
**Pull request by CJ-Rose**
includes updated design for ReportsHeader
- category tags moved out of the filter
- tags, search, filter, sort moved to one line on large screen, stacked on mobile
- filter opens Drawer on mobile, Dialog on desktop

*also updates Badge style for ReportCard and Actions route components after Badge variants changed in #27

### CODE REVIEW

File: app/components/reports/reports-filters.tsx [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+						</div>
+					))}
+				</div>
+				<Separator
+					className="md:hidden bg-vd-blue-500 my-6"
+					orientation="horizontal"
+				/>
+				<Separator
+					className="hidden md:flex bg-vd-blue-500 mx-6"
+					orientation="vertical"
```
Comment (CJ-Rose) [inline app/components/reports/reports-filters.tsx:67]: can anyone spot what's wrong here? I haven't been able to get a vertical separator to show up, with this component or with a `<div classname="w-[1px] h-full"/>`

Comment (thebeyondr) [reply app/components/reports/reports-filters.tsx:67]: Wrap both in a flex div and it should work. 

I think the design should account for more than five options and selecting multiple at once. This component is usually called a "Multiselect". Please investigate!

Comment (CJ-Rose) [reply app/components/reports/reports-filters.tsx:67]: > I think the design should account for more than five options and selecting multiple at once.

not sure what you mean by more than 5 options.. are you referring to the longer list of states as reports are added?

Comment (thebeyondr) [reply app/components/reports/reports-filters.tsx:67]: > > I think the design should account for more than five options and selecting multiple at once.
> 
> not sure what you mean by more than 5 options.. are you referring to the longer list of states as reports are added?

yes, that general rule applies to any other input types that apply.

- less than 5 options: radio / checkbox
- 5 or more options: dropdown <-- VD will most likely fit here

**Comment by CJ-Rose** [comment]
@thebeyondr review and merge this new layout as is or want to wait until i get everything hooked up?

File: app/components/reports/reports-filters.tsx [hunk 2]
```diff
@@ -46,24 +47,31 @@ const FilterItems: React.FC<ReportFiltersProps> = ({
...
 			<div className="flex flex-col md:flex-row md:justify-between">
 				<div className="pl-4">
-					<h2 className="font-medium pb-2 md:pb-4">Story from</h2>
+					<h2 className="text-lg font-medium pb-2 md:pb-4">Story from</h2>
 					{outlets.map((outlet: string) => (
-						<div key={outlet} className="flex items-center gap-2 pb-1">
-							<Circle size={18} strokeWidth={1} />
-							<p className="text-xs">{outlet}</p>
+						<div key={outlet} className="flex items-center gap-2 pb-2">
+							<Checkbox className="h-6 w-6 rounded-full border-vd-blue-500 data-[state=checked]:bg-vd-blue-500 data-[state=checked]:text-vd-beige-100" />
```
Comment (thebeyondr) [inline app/components/reports/reports-filters.tsx]: Let's not style these like radio buttons, but instead keep them "boxy" like:
image

**Review by thebeyondr — CHANGES_REQUESTED** [review]
LGTM, you can update the checkbox

**Review by thebeyondr — APPROVED** [review]
fuck it we ball

**Pull request merged**